### DISCUSSION TITLE
off device/site backup

### DISCUSSION THREAD

**Discussion by GigaTech** [home-assistant/frontend#9014] [Supervisor] [answered]
Hi Guys,
We really need a good official solution to get the (automated) backups automatically off of the device.

I know this is a can of worms with the options for 'destinations' but something really does needs to be done. I think a good clean start might be to NAS or other network share requiring ip/dns name, u/n and p/w easily configurable from the ui

I strongly feel this is more than a request, it is more of a requirement of any device or appliance with complex configs and custom scripts, automations and logging history.

There are many discussions on this topic in various threads all with unsupported and or incomplete examples, but I would only trust an official route on this important and necessary issue.

Many thanks guys/gals keep up the good work ;)

**Comment by frenck** [comment, marked as answer]
Hi there @GigaTech!

Thanks for opening a discussion at the frontend, however, you are placing a general feature request that is not frontend specific.

If you want to suggest a feature, you should try our [Community Forum: Feature Requests](https://community.home-assistant.io/c/feature-requests).

If you have additional questions, feel free to join our [Discord chat server](https://discord.gg/c5DvZ4e).

Thanks! 👍
